src/policy/fees.cpp

```diff
@@ -343,10 +336,8 @@ void CBlockPolicyEstimator::processTransaction(const CTxMemPoolEntry& entry, boo
 
 void CBlockPolicyEstimator::processBlockTx(unsigned int nBlockHeight, const CTxMemPoolEntry& entry)
 {
-    if (!entry.WasClearAtEntry()) {
-        // This transaction depended on other transactions in the mempool to
-        // be included in a block before it was able to be included, so
-        // we shouldn't include it in our calculations
+    if (!removeTx(entry.GetTx().GetHash())) {
```

Comment:
Does calling this here result in one extra call to `removeTx()` for each in-mempool, in-block transaction?  It looks like we also will call this in `CTxMempool::removeUnchecked`, which I guess is harmless, but worth commenting.